### PR TITLE
feat(shareable-lists): create and bulk add to shareable list

### PR DESCRIPTION
**Pull request by kschelonka**
Create a new list and bulk-add to it in one mutation.

[POCKET-9504]

[POCKET-9504]: https://mozilla-hub.atlassian.net/browse/POCKET-9504?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: servers/shareable-lists-api/src/database/mutations/ShareableList.ts [hunk 1]
```diff
@@ -101,6 +102,100 @@ export async function createShareableList(
...
+    return { items, list };
+  });
+  const completeList: ShareableList = {
+    ...list,
+    // bigint causes issues again...
+    userId: list.userId as unknown as bigint,
+    listItems: items as any, // nested bigints
+  };
+  //send event bridge event for shareable-list-created event type
+  sendEventHelper(EventBridgeEventType.SHAREABLE_LIST_CREATED, {
```
Comment (bassrock) [inline servers/shareable-lists-api/src/database/mutations/ShareableList.ts:180]: _in the future_ we will prob want or need a bulk add event.

File: servers/shareable-lists-api/schema-public.graphql [hunk 2]
```diff
@@ -256,4 +256,11 @@ type Mutation {
     listExternalId: ID!
     items: [AddItemInput!]! @constraint(minItems: 1, maxItems: 30)
   ): ShareableList!
+  """
+  Add a batch of items to an existing shareable list.
+  """
+  createAndAddToShareableList(
+    listData: CreateShareableListInput!
+    itemData: [AddItemInput!]! @constraint(minItems: 1, maxItems: 30)
```
Comment (bassrock) [inline servers/shareable-lists-api/schema-public.graphql:264]: 😍 that directive. Thing of amazement.

**Review by bassrock — APPROVED** [review]
lgtm!

File: servers/shareable-lists-api/src/database/mutations/ShareableList.ts (hunk 1 above)
Comment (kschelonka) [reply servers/shareable-lists-api/src/database/mutations/ShareableList.ts:180]: Yes

File: servers/shareable-lists-api/schema-public.graphql (hunk 2 above)
Comment (kschelonka) [reply servers/shareable-lists-api/schema-public.graphql:264]: Isn't it?!

**Pull request merged**